scraping/scraping/start_scripts/zalando_de.py

```diff
@@ -34,34 +34,54 @@ def combine_results(
 
 def male() -> List[dict]:
     path_2_category = {
-        "herrenbekleidung-shirts": "SHIRT",
-        "herrenbekleidung-hemden": "SHIRT",
-        "herrenbekleidung-sweatshirts-hoodies": "SWEATER",
-        "herrenbekleidung-pullover-strickjacken": "SWEATER",
-        "herrenbekleidung-jacken": "JACKET",
-        "herrenbekleidung-maentel": "JACKET",
-        "herrenbekleidung-anzuege": "SUIT",
-        "herrenbekleidung-jeans": "JEANS",
-        "herrenbekleidung-hosen": "PANT",
-        "herrenbekleidung-hosen-shorts": "PANT",
-        "herrenbekleidung-trainingsanzuege-jogger": "TRACKSUIT",
-        "herrenbekleidung-waesche": "UNDERWEAR",
-        "herrenbekleidung-nachtwaesche": "NIGHTWEAR",
-        "herrenbekleidung-bademode": "SWIMMWEAR",
-        # Shoes and Bags
-        "herrenschuhe": "SHOES",
-        "taschen-accessoires-taschen-herren": "BAG",
-        # Sport
-        "sports-herren-shirts": ("SHIRT", {"type": "SPORT"}),
-        "sports-herren-jacken": ("JACKET", {"type": "SPORT"}),
-        "sporthosen-herren": ("PANT", {"type": "SPORT"}),
-        "sports-herren-pullover-sweater": ("SWEATER", {"type": "SPORT"}),
-        "trainingsanzug-herren": ("TRACKSUIT", {"type": "SPORT"}),
-        "funktionsunterwaesche-herren": ("UNDERWEAR", {"type": "SPORT"}),
-        "sports-herren-struempfe": ("UNDERWEAR", {"type": "SPORT"}),
-        "sports-bademode-herren": ("SWIMMWEAR", {"type": "SPORT"}),
-        "sportschuhe-herren": ("SHOES", {"type": "SPORT"}),
-        "sports-taschen-rucksaecke-herren": ("BAG", {"type": "SPORT"}),
+        "/herrenbekleidung-shirts-basic/": ProductCategory.TSHIRT.value,
+        "/herrenbekleidung-shirts-print/": ProductCategory.TSHIRT.value,
+        "/herrenbekleidung-shirts-tops/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-shirts-poloshirts/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-shirts-longsleeves/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-shirts-sport/": (ProductCategory.TSHIRT.value, {"type": "SPORT"}),
+        "/herrenbekleidung-hemden/": ProductCategory.SHIRT.value,
+        "/herrenbekleidung-sweatshirts-hoodies/": ProductCategory.SWEATER.value,
+        "/herrenbekleidung-strickpullover/": ProductCategory.SWEATER.value,
+        "/herrenbekleidung-strickjacken/": ProductCategory.JACKET.value,
+        "/herrenbekleidung-jacken/": ProductCategory.JACKET.value,
+        "/herrenbekleidung-maentel/": ProductCategory.JACKET.value,
+        "/herrenbekleidung-anzuege/": ProductCategory.SUIT.value,
+        "/herrenbekleidung-jeans/": ProductCategory.JEANS.value,
+        "/herrenbekleidung-hosen/": ProductCategory.PANTS.value,
+        "/herrenbekleidung-hosen-shorts/": ProductCategory.SHORTS.value,
+        "/herrenbekleidung-trainingsanzuege-jogger/": ProductCategory.TRACKSUIT.value,
```

Comment:
This one includes "Trainingsanzüge",  "Jogginghosen",  "Trainingsjacken". 
can we split it into these subcategories and assign the corresponding categories (`TRACKSUIT, PANTS, JACKET`)? for `PANTS  and `JACKET` maybe with additional attribute `SPORTS`?